test.sh

```diff
@@ -15,6 +15,5 @@ python -m pip install \
   -r requirements.txt
 if ! [ -f test.ext4 ] || ! [ -f test.ext4.tmp ];then
   ./_test_image.sh
-  trap "rm -f test.ext4{,.tmp}" EXIT
```

Comment:
This should not be removed. The test cleans up after itself if the rest artifacts did not exist initially, and will not clean up if they already existed.

Reply:
done. I didn't notice the trap at first.